spec/requests/partner_requests_spec.rb

```diff
@@ -0,0 +1,96 @@
+require "rails_helper"
+
+RSpec.describe "Partner Requests Controller", type: :request do
+  context "when user authenticated" do
+    let!(:partner) { create(:partner, :verified) }
+    let!(:user) { create(:user, partner: partner) }
+
+    before do
```

Comment:
I prefer to avoid `before` blocks like this for the same reasons that I like to avoid `let`